roles/awscommon/tasks/vpc-facts.yml

```diff
@@ -8,3 +8,43 @@
   register: vpc_instance_facts
   tags:
     - always
+
+- name: get VPC data
+  ec2_vpc_net_facts:
+    filters:
+      "tag:Stack": "{{ vpc_name }}"
+  register: vpc_data
+  tags:
+    - always
+
+- name: Set VPC id
+  set_fact:
+    vpc_id: "{{ vpc_data.vpcs.0.id }}"
```

Comment:
can you add an example of what you get back on that vpc_data.vpcs array ?

Reply:
do you always want the first element?

Reply:
Sure, wrt example and yes, we get back an array with one element